lavamoat/build-system/policy-override.json

```diff
@@ -142,6 +175,7 @@
         "typescript": true,
         "eslint-plugin-jest>@typescript-eslint/utils": true,
         "@typescript-eslint/eslint-plugin>@typescript-eslint/type-utils": true,
+        "@typescript-eslint/eslint-plugin>@typescript-eslint/utils": true,
         "@typescript-eslint/parser>@typescript-eslint/scope-manager": true,
         "@typescript-eslint/eslint-plugin>tsutils": true,
         "eslint>debug": true,
```

Comment:
Overrides I had to add when running `yarn lavamoat:auto` I was going to add all the screenshots to validate this but there were A LOT. So will add some of the warning this fixes

![Screen Shot 2022-08-24 at 4 26 12 PM](https://user-images.githubusercontent.com/8112138/186770501-0537914b-8c39-481f-b34f-f628bd86701a.png)
![Screen Shot 2022-08-24 at 4 35 10 PM](https://user-images.githubusercontent.com/8112138/186770508-94505195-eb8f-4792-b236-feb4a1748b4e.png)
![Screen Shot 2022-08-24 at 4 35 18 PM](https://user-images.githubusercontent.com/8112138/186770511-94a0846a-7ef6-4f3e-9b41-46ff049965be.png)
![Screen Shot 2022-08-24 at 4 51 15 PM](https://user-images.githubusercontent.com/8112138/186770517-54cc8efb-984e-4089-aaaa-8503ed884400.png)
![Screen Shot 2022-08-24 at 4 57 16 PM](https://user-images.githubusercontent.com/8112138/186770518-5e19a8c9-8ce0-4028-a69a-e66595535599.png)
![Screen Shot 2022-08-24 at 5 02 03 PM](https://user-images.githubusercontent.com/8112138/186770519-be6f89ed-7184-437a-b440-2f310f7a92a8.png)
![Screen Shot 2022-08-24 at 5 13 20 PM](https://user-images.githubusercontent.com/8112138/186770521-9944e98a-f919-4e59-8d1e-3e81e4a286d4.png)

Reply:
Ugh sorry to hear about this, I'm sure this took a long time to sort out 😖